activemq-client/src/main/java/org/apache/activemq/ActiveMQSession.java

```diff
@@ -30,34 +30,10 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import jakarta.jms.BytesMessage;
-import jakarta.jms.Destination;
+import jakarta.jms.*;
```

Comment:
I don't think we should use wildcard imports

Reply:
Yeah the IDE just made this change. So annoying I thought I turned it off.